src/shared/components/organisms/navbar/Navbar.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button } from 'antd';
+import cn from 'classnames';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { showDrawer } from '../../../../redux-features/common';
+import styles from './navbar.module.scss';
+
+export interface INavbarItem {
+  name: string;
+  url: string;
+}
+
+export interface INavbar {
+  isUserLoggedin: boolean;
+  navbarItemList: Array<INavbarItem>;
+}
+
+const Navbar: React.FC<INavbar> = ({ isUserLoggedin, navbarItemList }) => {
+  const dispatch = useDispatch();
+
+  const handleButtonClick = () => {
+    if (isUserLoggedin) {
```

Comment:
- if-else can be avoided with early returns
- Need to have separate handlers for each button